### PR TITLE
.pullapprove.yml: remove

### PR DESCRIPTION
**Pull request by kolyshkin**
We have switch to native github approval mechanism
(https://github.com/opencontainers/selinux/issues/138),
so remove this.

(this PR also aims to test the new "required checks" and lgtm/merge settings)

### CODE REVIEW

**Review by giuseppe — APPROVED** [review]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
Worked! Merge was blocked until I approved 👍

**Pull request merged**
